src/dotnet/APIView/APIViewWeb/Helpers/PageModelHelpers.cs

```diff
@@ -334,11 +334,12 @@ public static int ComputeActiveConversationsInActiveRevision(CodeLine[] lines, R
                 Review = review,
                 Navigation = activeRevisionRenderableCodeFile.CodeFile.Navigation,
                 codeLines = codeLines,
-                APIRevisions = revisions.GroupBy(r => r.APIRevisionType).ToDictionary(r => r.Key.ToString(), r => r.ToList()),
-                ActiveRevision = activeRevision,
-                DiffRevision = diffRevision,
+                APIRevisionsGrouped = revisions.GroupBy(r => r.APIRevisionType).ToDictionary(r => r.Key.ToString(), r => r.ToList()),
+                ActiveAPIRevision = activeRevision,
+                LatestApprovedAPIRevision = revisions.Where(r => r.IsApproved).OrderByDescending(r => r.ChangeHistory.Last(ch => ch.ChangeAction == APIRevisionChangeAction.Approved).ChangedOn).First(),
+                DiffAPIRevision = diffRevision,
                 TotalActiveConversiations = comments.Threads.Count(t => !t.IsResolved),
-                ActiveConversationsInActiveReviewRevision = ComputeActiveConversationsInActiveRevision(activeRevisionHtmlLines, comments),
```

Comment:
I am lacking context here, but I am seeing how there was a rename from `ActiveRevision` to `ActiveAPIRevision`. Is this a significant different concept? 
I ask because in some cases functions changed name like `ActiveConversationsInActiveAPIRevision`. 
but other didn't like the one in this line. same with some variables like  `activeREvisionHtemLines`.

Reply:
The rename is not significant. Am just trying to us `apiRevision` instead of just `revisions` because there are also `sampleRevisions`. Not a big deal but I try to make the naming more specific as I encounter them.